pkgs/data/fonts/google-fonts/default.nix

```diff
@@ -39,7 +39,7 @@ stdenv.mkDerivation rec {
     dest=$out/share/fonts/truetype
     mkdir -p $dest
     find . -name "*.ttf" -exec cp -v {} $dest \;
-    chmod -x $dest/*.ttf
+    (cd $dest; chmod -x *.ttf)
```

Comment:
It might better to use `xargs` and the `install` command which can also set the executable bit directly.

Reply:
If we're golfing, lines 40-42 seem equivalent to something like
```
find . -name '*.ttf' -exec install -Dt $dest '{}' +
```

Reply:
Did not know about this.